src/main/java/gameboy/gamepalyer/GamePlayer.java

```diff
@@ -0,0 +1,19 @@
+package gameboy.gamepalyer;
+
+import gameboy.gamepack.*;
+import gameboy.gamepack.racinggame.RacingGamePack;
+
+public class GamePlayer {
+
+    public GamePlayer() { }
+
+    public void startGame(GamePack gamePack) {
+        gamePack.play();
+    }
+
+    public static void main(String[] args) {
+        GamePlayer gamePlayer = new GamePlayer();
+        gamePlayer.startGame(new RacingGamePack());
+    }
```

Comment:
게임을 시작하는 main 클래스네요

```suggestion
    public static void main(String[] args) {
        RacingGamePack racingGamePack = new RacingGamePack();
        racingGamePack.play();
    }
```

GamePlayer를 인스턴스화 한 이유가 있나요?
제가 제안한 코드와 차이가 없어보여요 🤔

Reply:
main이 GamePlayer 클래스에서 분리되어야하나? 하는 생각에 넣었는데 지금은 없는게 나아 보이네요